smarctlide-monitoring/src/test/resources/services-config.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="utf-8"?>
+<config xmlns="http://www.atb-bremen.de" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
```

Comment:
Files in `src/test/resources` don't seem to be needed because there are no test cases

Reply:
not sure whether we should leave them as examples or remove them

Reply:
Currently, they are not working as examples because they reference `de.atb.context.monitoring.parser.file.DummyFilePairParser` and `de.atb.context.monitoring.analyser.file.DummyFilePairAnalyser`, which are located under `src/test/`, are thus not exported and cannot be instantiated in `de.atb.context.monitoring.config.models.InterpreterConfiguration.createParser` (line 219) - causing a `ClassNotFoundException`.

The main branch should also be merged into this branch (or this branch should be rebased onto main), so that the fix for writing the index segments file is included.

Reply:
rebase done.

Reply:
unused config files removed

Reply:
The test config files in [smartclide-monitoring/src/test/resources](https://github.com/eclipse-researchlabs/smartclide-context/tree/dockerization/smartclide-monitoring/src/test/resources) are still there.

But the ones in [context-monitoring/src/test](https://github.com/eclipse-researchlabs/smartclide-context/tree/dockerization/context-monitoring/src/test) have been removed. I think those are actually needed for the tests in `context-monitoring`?

BTW, it seems that those tests are not run anymore?

```
[INFO] --- maven-surefire-plugin:3.0.0-M5:test (default-test) @ context-monitoring ---
[INFO] 
[INFO] -------------------------------------------------------
[INFO]  T E S T S
[INFO] -------------------------------------------------------
[INFO] 
[INFO] Results:
[INFO] 
[INFO] Tests run: 0, Failures: 0, Errors: 0, Skipped: 0
```

Reply:
test configs fixed.

Reply:
The reason why the tests are not running is that spring-boot-test uses junit5 by default. But the test cases still use junit4.

To fix it add the following dependency to `context-monitoring/pom.xml`:

```
<dependency>
    <groupId>org.junit.vintage</groupId>
    <artifactId>junit-vintage-engine</artifactId>
    <scope>test</scope>
    <exclusions>
        <exclusion>
            <groupId>org.hamcrest</groupId>
            <artifactId>hamcrest-core</artifactId>
        </exclusion>
    </exclusions>
</dependency>
```

See https://docs.spring.io/spring-boot/docs/2.4.2/reference/htmlsingle/#boot-features-testing